packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1026,7 +1063,8 @@ export class TransactionController extends BaseControllerV1<
 
     log('Submitting speed up transaction', { oldFee, newFee, txParams });
 
-    const hash = await query(this.ethQuery, 'sendRawTransaction', [rawTx]);
+    const ethQuery = this.getEthQuery(transactionMeta.networkClientId)
+    const hash = await this.publishTransaction(ethQuery, rawTx)
```

Comment:
usually we want to only submit transactions on the networkClientId that the user originally approved the request for, but in the case of cancelling it's okay that we don't use the exact same network client